composer.lock

```diff
@@ -7625,6 +7543,65 @@
             ],
             "time": "2024-04-05T04:35:58+00:00"
         },
+        {
+            "name": "rector/rector",
+            "version": "1.0.5",
+            "source": {
+                "type": "git",
```

Comment:
Still contains `rector`. Can we update this lockfile too, please.

Reply:
Of course - sorted. thanks.